hydrolib/core/dflowfm/ini/util.py

```diff
@@ -622,3 +622,45 @@ def rename_keys_for_backwards_compatibility(
                 break
 
     return values
+
+
+class UnknownKeywordErrorManager:
+    """
+    Error manager for unknown keys.
+    Detects unknown keys and manages the Error to the user.
+    """
+
+    def raise_error_for_unknown_keywords(
+        self,
+        data: Dict[str, Any],
+        section_header: str,
+        fields: Dict[str, ModelField],
+        excluded_fields: Set,
```

Comment:
Set[str]

Reply:
Agree, but... _exclude_fields from INIBaseModel returns a set, not a set[str], it is a set[str], but the typehint only suggests a set.

Reply:
done